### PR TITLE
add guard against undefined judge

### PR DESCRIPTION
**Pull request by youngfreezyVA**
Fixes https://sentry.ds.va.gov/department-of-veterans-affairs/caseflow/issues/4186/events/281279/

### CODE REVIEW

File: client/app/queue/SubmitDecisionView.jsx [hunk 1]
```diff
@@ -133,7 +133,9 @@ class SubmitDecisionView extends React.PureComponent {
       type: checkoutFlow === DECISION_TYPES.DRAFT_DECISION ?
         'decision' : 'outside medical opinion (OMO) request',
       veteran: veteranFullName,
-      judge: judges[decision.opts.reviewing_judge_id].full_name
+      judge: judges[decision.opts.reviewing_judge_id] ?
+        judges[decision.opts.reviewing_judge_id].full_name :
+        `${this.props.task.assignedBy.first_name} ${this.props.task.assignedBy.last_name}`
```
Comment (aroltsch) [inline client/app/queue/SubmitDecisionView.jsx]: will task always exist here?

Comment (youngfreezyVA) [reply client/app/queue/SubmitDecisionView.jsx]: there's a chance it's not defined. i'll add a check for that

Comment (youngfreezyVA) [reply client/app/queue/SubmitDecisionView.jsx]: done

File: client/app/queue/SubmitDecisionView.jsx [hunk 2]
```diff
@@ -147,6 +147,19 @@ class SubmitDecisionView extends React.PureComponent {
       });
   };
 
+  getJudgeValueForSuccessMessage = (judges, decision) => {
+    const judgeIsInJudgesArray = judges[decision.opts.reviewing_judge_id];
+
+    if (judgeIsInJudgesArray) {
+      return judgeIsInJudgesArray.full_name;
+    }
+    if (this.props.task) {
```
Comment (aroltsch) [inline client/app/queue/SubmitDecisionView.jsx]: Let's check for an existence of assigned_by as well. 
```
if (this.props.task && this.props.task.assignedBy) {}
```

**Pull request merged**